src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/CallingConvention.cs

```diff
@@ -11,5 +11,6 @@ public enum CallingConvention
         StdCall = 3,
         ThisCall = 4,
         FastCall = 5,
+        SwiftCall = 6,
```

Comment:
This should be reverted. We are not adding new calling conventions to this legacy enum.

Reply:
Yes, reverted.